benchmarks/benchmarks/optimize_milp.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+import numpy as np
+from numpy.testing import assert_allclose
+
+from .common import Benchmark, safe_import
+
+with safe_import():
+    from scipy.optimize import milp
+
+with safe_import():
+    from scipy.optimize.tests.test_linprog import magic_square
+
+
+# MIPLIB 2017 benchmarks included with permission of the authors
+# The MIPLIB benchmark problem set was downloaded from https://miplib.zib.de/.
+# An MPS converter (scikit-glpk) was used to load the data into Python. The
+# arrays were arranged to the format required by `milp` and saved to `npz`
+# format using `np.savez`.
+milp_problems = ["piperout-27"]
+
+
+class MilpMiplibBenchmarks(Benchmark):
+    params = [milp_problems]
+    param_names = ['problem']
+
+    def setup(self, prob):
```

Comment:
Here and in the other benchmark, is it really better to have the parameter name different from the name in `param_names`?

Maybe it is just habit for me, but I usually just use the same names that I put in the `param_names` list, which also show up in the benchmark output I think.

Reply:
No. I prepared this by copy-pasting and not paying too much attention to it.